build/Dockerfile

```diff
@@ -59,7 +59,16 @@ RUN --mount=type=secret,id=nginx-repo.crt,dst=/etc/ssl/nginx/nginx-repo.crt,mode
 	&& rm -rf /var/lib/apt/lists/*
 
 
-############################################# Base image for Debian with NGINX Plus and App Protect WAF/DoS #############################################
```

Comment:
Do we need this commented out part?

Reply:
I think there's still something left in line 62